docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -19,29 +23,35 @@ This is similar to setting `--channel=latest/stable` or shorthand `--stable` and
 
-For device services, the  Device and Device Profile files are submitted to [Core Metadata](../../microservices/core/metadata/Ch-Metadata) upon initial startup. Refer to the documentation of [Device Services](../../microservices/device/Ch-DeviceServices/) for details.
+For device services, the Device and Device Profile files are submitted to [Core Metadata](../../microservices/core/metadata/Ch-Metadata) upon initial startup. Refer to the documentation of [Device Services](../../microservices/device/Ch-DeviceServices/) for details.
 
 #### Config registry
-The configurations that are uploaded to Consul can be modified using Consul's UI or [kv REST API](https://www.consul.io/api/kv). Changes to configurations in Consul are loaded by the service at startup. If the service has already started, a restart is required to load new configurations. Configurations that are in the writable section get loaded not only at startup, but also during the runtime. In other words, changes to the writable configurations are loaded automatically without a restart. Please refer to 
-[Common Configuration](/microservices/configuration/CommonConfiguration/) and 
+The configurations that are uploaded to the Registry (i.e. Consul by default) can be modified using Consul's UI or [kv REST API](https://www.consul.io/api/kv). The Registry is a Core services, part of the [Platform Snap](#platform-snap).
+
+Changes to configurations in Registry are loaded by the service at startup. If the service has already started, a restart is required to load new configurations. Configurations that are in the writable section get loaded not only at startup, but also during the runtime. In other words, changes to the writable configurations are loaded automatically without a restart.
+
+Please refer to 
+[Common Configuration](../../microservices/configuration/CommonConfiguration/) and 
 [Configuration and Registry Providers](../../microservices/configuration/ConfigurationAndRegistry/) for more information.
 
 #### Config provider snap
 [config-provider-snap]: #config-provider-snap
 
-Most EdgeX snaps have a [content interface](https://snapcraft.io/docs/content-interface) which allows another snap to seed the snap with configuration files.
-This is useful when replacing entire configuration files via another snap, packaged with the deployment-specific configurations.
+Most EdgeX snaps have a [content interface](https://snapcraft.io/docs/content-interface) which allows another snap to seed it with configuration files.
+This is useful for replacing all the configuration files in a service snap via a config provider snap without manual user interaction.
+
+A config provider snap could be a standalone package with all the necessary configurations for multiple snaps. It will expose and one or more [interface](https://snapcraft.io/docs/interface-management) *slots* to allow connections from consumer *plugs*. The config provider snap can be released to the store just like any other snap. 
```

Comment:
> "The config provider snap can be released to the store.... "
Is this snapstore or brandstore?

Reply:
It can be published to either of them. 

The store: https://snapcraft.io/store
Or any IoT app store: https://ubuntu.com/internet-of-things/appstore

https://ubuntu.com/core/services/guide/snap-store-vs-iot-app-store